### PR TITLE
Improved docs first index page

### PR DESCRIPTION
**Pull request by keeeto**
**Description of work.**

This has been adjusted to be an improvement to the first index page on loading the documentation, to make it more user focussed and clearer where novice users can go for help.

**To test:**

* Build the `docs-html` target of Mantid. I found that on OSX I had to build a regular target first then build the `docs-html` target
* Check the front index page of documentation 
* Check that the text makes sense

Does this update require release notes?
- [ ] Yes
- [x] No

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/basics/introduction.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    * :ref:`training`, with links to further learning resources.
+    * :ref:`concepts contents`, with deeper background information on aspects of Mantid.
+    * :ref:`interfaces contents`, with details of the bespoke technique interfaces available.
+    * :ref:`fitting contents`, with a guide to the science and art of fitting functions to data.
+    * :ref:`techniques contents`, with further information relevant to some of the specific scientific techniques.
+    * :ref:`Mantid Matplotlib Plot Gallery and Examples <plotting>`, with details of how to make pretty plots.
+
+
++ In addition to documentation we also have
+    * A users help forum: `Here <http://forum.mantidproject.org/>`_.
```
Comment (peterfpeterson) [inline docs/source/basics/introduction.rst]: Change this line to
```
* A `forum <http://forum.mantidproject.org/>`_.
```

File: docs/source/basics/introduction.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    * A Mantid help email address: mantid-help@mantidproject.org.
```
Comment (peterfpeterson) [inline docs/source/basics/introduction.rst]:
```
* An email address: `mantid-help@mantidproject.org <mailto:mantid-help@mantidproject.org>`__.
```

File: docs/source/basics/loading_and_displaying_data.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+The "Load" feature can handle any of these formats automatically.
+
+Multiple Data Files
+###################
+
+You can load multiple files into mantid with a single Load command, either keeping each workspace separate, 
+or summing the data into a single workspace:
+
+* <name>, <name> : load a list
```
Comment (peterfpeterson) [inline docs/source/basics/loading_and_displaying_data.rst]: escape these

File: docs/source/basics/loading_and_displaying_data.rst [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+* Simulated data formats.
+* Ascii data, Table data, etc.
+* Live data streams.
+
+The "Load" feature can handle any of these formats automatically.
+
+Multiple Data Files
+###################
+
+You can load multiple files into mantid with a single Load command, either keeping each workspace separate,
```
Comment (peterfpeterson) [inline docs/source/basics/loading_and_displaying_data.rst]: link to http://docs.mantidproject.org/nightly/api/python/mantid/api/MultipleFileProperty.html

File: docs/source/basics/loading_and_displaying_data.rst [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+* Navigating to the "File->Load->File" menu item.
+* Clicking the |load_symbol| symbol in the toolbar.
+* Using the "Load" algorithm. (:ref:`load algorithm`).
+
+Types of Data Files
+###################
+
+Mantid can load many different data formats. A few important examples are:
+
+* ISIS, SNS, ILL, PSI .nexus data files.
```
Comment (peterfpeterson) [inline docs/source/basics/loading_and_displaying_data.rst]: escape file extensions

**Review by peterfpeterson — CHANGES_REQUESTED** [review]
Lots of changes requested. This feels like a PR that should be marked "in progress."

File: docs/source/basics/algorithms_workspaces_and_history.rst [hunk 6]
```diff
@@ -0,0 +1,50 @@
+.. _algorithms workspaces and history:
+
+================================================
+Understanding algorithms, workspaces and history
```
[the remark this answers is not in the document]
Comment (keeeto) [reply docs/source/basics/algorithms_workspaces_and_history.rst]: @peterfpeterson agree on the non-necessity of history here. The reason for not folding into the concepts page is that this is intended as a very basic intro, just giving a brand new user what they need to get going. I think that it is pretty impossible to use Mantid without knowing what a workspace/algorithm is. The way I see it concepts pages provide deeper level understanding for the interested user. The basics docs just tell you what you need to know to get going.

**Comment by keeeto** [comment]
@peterfpeterson - thanks for the detailed review. I think that documentation is an area that should be somewhat developed collaboratively, so I appreciate the feedback and any criticisms are welcome (within reason :)).

File: docs/source/algorithms/Load-v1.rst [hunk 7]
```diff
@@ -6,6 +6,8 @@
 
 .. properties::
 
+.. _load algorithm:
```
[the remark this answers is not in the document]
Comment (keeeto) [reply docs/source/algorithms/Load-v1.rst]: `Load-v1.rst` previously had no label for linking as far as I can see. This is linked to in the 'loading and displaying data' guide. The link seems to work fine when I build it. Should I put the label elsewhere in the `.rst` file?

[the remark this answers is not in the document]
Comment (peterfpeterson) [reply docs/source/algorithms/Load-v1.rst]: All of the algorithm links are done automagically in the `.. algorithm::` directive. The link for `Load` is `:ref:`Load <algm-Load>` if you want to control the text.

File: docs/source/index.rst [hunk 8]
```diff
@@ -16,6 +16,7 @@ Mantid Documentation
    :glob:
    :maxdepth: 1
 
+   basics/index
```
Comment (NickDraper) [inline docs/source/index.rst]: This renders as "Mantid Basics"

A few lines later you have "Mantid Basics Course" referring to the training course.  These titles are too similar for two separate but related things.

Comment (NickDraper) [reply docs/source/index.rst]: Perhaps "Introduction to Mantid"?

File: docs/source/basics/introduction.rst [hunk 9]
```diff
@@ -0,0 +1,43 @@
+.. _mantid introduction:
+
+============
```
Comment (NickDraper) [inline docs/source/basics/introduction.rst]: This whole page has similar links to the top level index page, and might be better as a replacement of the top level index page.

File: docs/source/index.rst (hunk 8 above)
Comment (NickDraper) [reply docs/source/index.rst]: Also the index page itself is not a usefull index, it would be better to link to the Introduction page itself.

**Review by NickDraper — DISMISSED** [review]

File: docs/source/basics/loading_and_displaying_data.rst (hunk 4 above)
Comment (peterfpeterson) [reply docs/source/basics/loading_and_displaying_data.rst]:
```
:py:obj:`MultipleFileProperty <mantid.api.MultipleFileProperty>`
```

**Comment by NickDraper** [comment]
@peterfpeterson, or any other tester.

The only changes now are to the first into page to the documentation, all of the others were pointless copies of sections from the mantid basics course and have been removed.

Please check you are happy to have a link to monitor.sns.gov in the page, this can easily be removed if you are not happy.

**Review by peterfpeterson — APPROVED** [review]
Builds without warning and adds context to what the various sub-pages actually are.

**Pull request merged**